0it [00:00, ?it/s]1it [00:01,  1.04s/it]22it [00:01, 26.04it/s]48it [00:01, 59.94it/s]81it [00:01, 106.11it/s]116it [00:01, 153.42it/s]150it [00:01, 193.72it/s]187it [00:01, 234.18it/s]220it [00:01, 257.96it/s]253it [00:01, 261.78it/s]286it [00:01, 276.55it/s]317it [00:02, 270.24it/s]353it [00:02, 292.28it/s]388it [00:02, 306.12it/s]420it [00:02, 309.50it/s]452it [00:02, 305.39it/s]484it [00:02, 307.60it/s]516it [00:02, 310.23it/s]548it [00:02, 303.49it/s]581it [00:02, 310.09it/s]620it [00:03, 331.20it/s]654it [00:03, 318.47it/s]692it [00:03, 334.36it/s]731it [00:03, 349.67it/s]767it [00:03, 349.63it/s]803it [00:03, 342.27it/s]838it [00:03, 321.95it/s]878it [00:03, 341.31it/s]915it [00:03, 347.41it/s]947it [00:04, 236.50it/s]
Traceback (most recent call last):
  File "/usr/local/bin/jobinfo.py", line 14, in <module>
    gpu_stats = getstats(sys.argv[1])
  File "/usr/local/bin/jobinfo.py", line 10, in getstats
    with open(fname) as stats_file:
IOError: [Errno 2] No such file or directory: '/run/gpustats.json'
